src/components/aboutUs/about_utils/about.css

```diff
@@ -0,0 +1,19 @@
+.blank_nav{

```

Comment:
*delete this file and copy this whole css inside scss/_about.scss > .about-us { HERE }